pkg/contract/private_key.go

```diff
@@ -47,13 +59,17 @@ func GetPrivateKeyFromFlags(
 	app *application.Avalanche,
 	flags PrivateKeyFlags,
 	genesisPrivateKey string,
+	mutuallyExclusiveErrMsg string,
 ) (string, error) {
+	if mutuallyExclusiveErrMsg == "" {
+		mutuallyExclusiveErrMsg = "--private-key, --key and --genesis-key are mutually exclusive flags"
```

Comment:
also we hardcode flag strings in the message but provide `AddPrivateKeyFlagsToCmd` flexibility to have custom flag strings